### PR TITLE
Forgot Password UI

### PR DESCRIPTION
**Pull request by Stokalas**
Simple prompts and form to allow user to change password.
**Files to look at:** 
- Prompt.cs (taken from the web)
- FormChangePass.cs
- FormLogIn.cs

### CODE REVIEW

**Comment by MatasLa** [comment]
Basic UI functionality, but doing the job as it should. Tested and everything seems to work fine.

File: SmartSaver/forms/FormLogIn.cs [hunk 1]
```diff
@@ -42,27 +40,41 @@ private void SetQuote()
...
-
-        private void linkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
-        {
-
-        }
-
         private void FormLogIn_Load(object sender, EventArgs e)
         {
 
         }
```
Comment (ciliakas) [inline SmartSaver/forms/FormLogIn.cs:46]: Is this supposed to be here?

**Review by ciliakas — COMMENTED** [review]
Tested it out, works as it supposed to, only minor things I noticed was the tab order in the ChangePassForm is wrong, also the form appears in the side of the screen, personally I change this setting to center screen. 
Possibly at this point it would make sense to move the forms associated with authorization to a seperate folder.

**Review by ciliakas — APPROVED** [review]
Everything looks good now. Hope you don't mind, I quickly deleted one line of code from the designer which was causing compiling issues (it was from the empty method which was deleted) and renamed the folders and namespaces according to convertion :)

**Pull request merged**